### PR TITLE
increase most read fresh data threshold to 60mintues

### PR DESCRIPTION
**Pull request by thekp**
Resolves #NUMBER

**Overall change:**
_Increase the most read fresh data threshold to 60minutes, our server logging has shown that we are consistently getting stale data. For the time being, this will hopefully reduce the number of users not seeing an empty page whilst we investigate why we might be receiving stale data._

**Code changes:**

- _update `MAXIMUM_STALE_DATA_TIME` to 60minutes_
- _update unit tests_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by sareh — APPROVED** [review]
👍 Looks good. 
Please also update the Run Book when this is merged in, so all references to '35 minutes' are updated there, too.

**Review by EinsteinNjoroge — COMMENTED** [review]
Shouldn't we update this as well?
https://github.com/bbc/simorgh/blob/9a706d2a9b5ec2e2aa044d6d88793af8cb1643ca/src/app/containers/MostRead/utilities/processMostRead.js#L98

File: src/app/containers/MostRead/utilities/index.test.js [hunk 1]
```diff
@@ -10,13 +10,13 @@ const calcTimestampDaysAgo = days =>
...
 
 describe('mostReadRecordIsFresh', () => {
-  it('should return true if 35 minutes ago or less', () => {
+  it('should return true if 60 minutes ago or less', () => {
     expect(mostReadRecordIsFresh(currentTime.toUTCString())).toEqual(true);
-    expect(mostReadRecordIsFresh(calcTimestampMinutesAgo(34))).toEqual(true);
+    expect(mostReadRecordIsFresh(calcTimestampMinutesAgo(59))).toEqual(true);
   });
 
   it('should return false if more than 35 minutes ago', () => {
```
Comment (tochwill) [inline src/app/containers/MostRead/utilities/index.test.js]:
```suggestion
  it('should return false if more than 60 minutes ago', () => {
```

**Review by tochwill — COMMENTED** [review]
Looking good! Just a tiny update in one of the test descriptions

**Review by tochwill — APPROVED** [review]

**Comment by thekp** [comment]
Unit tests cover the functionality.

**Pull request merged**